astro/src/diagrams/quickstarts/login-after-sdk.astro

```diff
@@ -0,0 +1,21 @@
+---
+import Diagram from "../../components/mermaid/Diagram.astro";
+const { alt } = Astro.props;
+
+//language=Mermaid
+const diagram = `
+sequenceDiagram
+    participant User
+    participant App as Application
+    participant FusionAuth
+
+    User ->> App : View Homepage
+    User ->> App : Click Login Link (to FusionAuth)
+    User ->> FusionAuth : View Login Form
+    FusionAuth ->> User : Show Login Form
+    User ->> FusionAuth : Fill Out and Submit Login Form
+    FusionAuth ->> FusionAuth : Authenticates User
+    FusionAuth ->> App : User is Authenticated
+    App ->> User : Display User's Account or Other Info`;
+---
```

Comment:
Please add a redirect and remove the existing react asciidoc quickstart, like we did here: https://github.com/FusionAuth/fusionauth-site/pull/2443/files

(see the change to src/cloudfront/fusionauth-website-request-handler.js for an example).

Reply:
I'll do it when the things above are resolved.

Reply:
I've changed the paths in:
- `site/docs/v1/tech/shared/_framework-integration-tutorial-list.adoc`
- `site/docs/quickstarts/index.html`
- `site/_layouts/doc.liquid`
- `src/cloudfront/fusionauth-website-request-handler.js`


I'm not sure about the following files:
- `site/_data/exampleapps.yaml`
- `astro/src/pages/docs/quickstarts/quickstart-sections.ts`
- `astro/.astro/types.d.ts` 

@mooreds there are several sections about React, can you maybe do this changes? 
I don't want to break things.. :)

Reply:
Sure, I'll do the changes.